tests/__sf_section_dotnet.test.fish

```diff
@@ -0,0 +1,158 @@
+source $DIRNAME/spacefish_test_setup.fish
+source $DIRNAME/mock.fish
+
+function setup
+	spacefish_test_setup
+	mock dotnet 0 "echo \"2.1.403\""
+	mkdir -p /tmp/tmp-spacefish
+	cd /tmp/tmp-spacefish
+end
+
+function teardown
+	rm -rf /tmp/tmp-spacefish
+end
+
+test "Prints nothing when required files are missing"
+	(
+		rm -f /tmp/tmp-spacefish/project.json
+		rm -f /tmp/tmp-spacefish/global.json
+		rm -f '/tmp/tmp-spacefish/*.sln'
+		rm -f '/tmp/tmp-spacefish/*.csproj'
+		rm -f '/tmp/tmp-spacefish/*.xproj'
+	) = (__sf_section_dotnet)
+end
+
+test "Prints section if project.json is present"
+	(
+		touch /tmp/tmp-spacefish/project.json
```

Comment:
nit: I would add a newline separating the setup from the test output like in other tests in this file 😀

Reply:
Done.